util-taglib/src/com/liferay/taglib/aui/ATag.java

```diff
@@ -60,7 +60,7 @@ protected int processEndTag() throws Exception {
 				jspWriter.write(StringPool.SPACE);
 				jspWriter.write("<svg class=\"lexicon-icon ");
 				jspWriter.write("lexicon-icon-shortcut\" focusable=\"false\" ");
-				jspWriter.write("role=\"img\"><use data-href=\"");
+				jspWriter.write("role=\"img\"><use href=\"");
```

Comment:
Loving this!

To make this foolproof, I was thinking that maybe this should be:

```
String hrefAttr = "href";

if (CDN) hrefAttr = "data-href"; // Note, no need to distinguish IE here

jspWriter.write(...);
```

That being said, the only effect of this in CDN would be that we'd see a rejected request to the icons spritemap. We probably can't control all calling sites anyways, with people generating raw markup like there's no tomorrow, so I'm leaning towards just keeping it simple... 

Thoughts?

Reply:
One offender would be `Clay`... and I doubt we want to go add this everywhere in Clay :D

Reply:
@jbalsas so are you thinking we would still run svg4everybody with `attributeName: 'data-href',` at https://github.com/liferay-frontend/liferay-portal/pull/220/files#diff-541d787ebc908170523bf4b674fa67afL66?

If so, we would likely want to check IE as well since we run svg4everybody with at attr. Or the other option is to also check for cdn where we instantiate svg4everybody

Reply:
> @jbalsas so are you thinking we would still run svg4everybody with attributeName: 'data-href', at https://github.com/liferay-frontend/liferay-portal/pull/220/files#diff-541d787ebc908170523bf4b674fa67afL66?

Yes, but never mind that. To really pull that off, we should make sure there's no `use href` anywhere in the whole page or contributed html... which I doubt we want. The only possible way to do this would be using a `filter` to rewrite the whole page before serving it. But filters are expensive... so I don't think this is worth it.

Just remember this in case it comes up in the future as something we might need to do :)

Reply:
Yeah thats what I was thinking, there are a handful of places that don't use the tag version and we would need it there as well. I'll add the other updates and try and measure some performance and then run tests to see where we are at with this.